### DISCUSSION TITLE
WiX Online Meeting #205

### DISCUSSION THREAD

**Discussion by robmen** [wixtoolset/issues#6346] [Announcements]
The next WiX Online meeting will be: February 4th, 2021 @9:30 Pacific

Agenda items:

* Triage
* WiX v4 Discussion
  * Major Upgrade table design flaw in Burn
  * #6176 - CustomTable/@BootstrapperApplicationData design
  * #4991 - Handling of related bundles missing from cache
  * #5950 - Rollback ExePackage on cancel
  * #4727 - Inline Directory Syntax on Component nested in ComponentGroup

Join us live: https://www.twitch.tv/firegiantco/
Join us later: https://www.youtube.com/c/wixtoolset/

**Comment by rseanhall** [comment]
If I only get time for one thing in the WiX v4 Discussion, I would prefer it to be reviewing https://github.com/wixtoolset/burn/pull/35.

**Comment by robmen** [comment]
Meeting highlights and video can be found at https://www.firegiant.com/blog/2021/2/4/wix-online-meeting-205-highlights/
